.npmignore

```diff
@@ -4,8 +4,4 @@ test/
 .eslintrc.js
 .eslintignore
 .gitignore
-hello.txt
-hello_parse.txt
-parse_hello.py
 demo
```

Comment:
we should remove this too.

Reply:
`demo` is still in the code base. I'm not sure if we want to include that in the npm module.